setup/purchase_discount/odoo/addons/purchase_discount

```diff
@@ -0,0 +1 @@
+../../../../purchase_discount
```

Comment:
This seems to be a regular file, instead of a symlink.
Due to this, the bot can't build a wheel and fails.

Why the bot does not report the error is unknown at this stage and tracked in https://github.com/OCA/oca-github-bot/issues/221

Reply:
@ramiadavid can you remove this file and run `pre-commit run --all-files` again?

Reply:
Hi @sbidoul
I work from Windows and pre-commit fails when it tries to create that file, I usually create it manually, but I don't know if it's correct.

Reply:
@ramiadavid ha, symlinks on windows may be complicated. I could push the fix to your branch.

Reply:
@sbidoul Thanks,
I'll check to see how I can make it work, for other migrations.